### PR TITLE
Improve folder structure for tests

### PR DESCRIPTION
**Pull request by its-tayo**
fixes issue #60

### CODE REVIEW

**Comment by ezkemboi** [comment]
Thanks for the PR @its-tayo. I will do a review. 
@hezronkimutai also, if you get some time,
you can take a look at this one.

File: README.md [hunk 1]
```diff
@@ -47,53 +50,62 @@ You can now create, drop any database models. The created model is **User**, und
...
+    │     ├─ controllers
+    │         ├─ index.js
+    │         ├─ user.js
+    │     ├─ middleware
+    │         ├─ index.js
+    │     ├─ routes
+    │         ├─ index.js
+    │     └─ index.js
+    │
     └── index.js
```
Comment (ezkemboi) [inline README.md]: I will fix this in the next PR. I think `index.js` should be inside `src/`

File: bin/createsrcdirandfiles.js [hunk 2]
```diff
@@ -1,169 +1,228 @@
...
 
-const createSrcDirAndFiles = (details) => {
-    const { appBaseDirectory, tests, database, orm } = details;
-    let foldersToAdd = ['src/controllers', 'src/routes', 'src/config', 'src/scripts', 'src/models', 'src/middlewares', 'src'];
-    if (tests) {
-        foldersToAdd.push('test');
+const createSrcDirAndFiles = details => {
+  const { appBaseDirectory, tests, database, orm } = details;
+  let foldersToAdd = [
+    "src/controllers",
```
Comment (ezkemboi) [inline bin/createsrcdirandfiles.js]: I think this is good and readable.

File: README.md (hunk 1 above)
Comment (its-tayo) [reply README.md]: Alright, thanks.

File: bin/createsrcdirandfiles.js (hunk 2 above)
Comment (its-tayo) [reply bin/createsrcdirandfiles.js]: @ezkemboi  Thanks, I think we should set up linting for this project, I could create an issue addressing this.

Comment (ezkemboi) [reply bin/createsrcdirandfiles.js]: https://github.com/kemboijs/nodejs-api-cli/issues/25

**Pull request merged**